.github/workflows/auto_cherry_pick.yml

```diff
@@ -0,0 +1,26 @@
+name: Auto Cherry Pick 
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  auto-cherry-pick:
+    if_merged:
+      if: ${{ startsWith(github.event.pull_request.labels.*.name, 'cherry-pick-') && github.event.pull_request.merged == true }}
+      runs-on: [self-hosted, nebula-fast]
+      container:
+        image: reg.vesoft-inc.com/vesoft/nebula-dev:centos7
```

Comment:
This docker image is used to build nebula source, I think you could try other thin image like `ubuntu` directly.